### PR TITLE
Add teams report button to groups screen

### PR DESCRIPTION
**Pull request by jondewijones**
### What

Add teams report button for downloadable teams report 

### Who can review

Anyone

### CODE REVIEW

File: directors/directors.go [hunk 1]
```diff
@@ -72,4 +72,11 @@ func setHeaders(req *http.Request) {
 			log.Event(req.Context(), "unable to set refresh token header", log.Error(err), log.ERROR)
 		}
 	}
+
+	if req.URL.Path == "/groups-report" {
+		if err := headers.SetAccept(req, "text/csv"); err != nil {
+			log.Event(req.Context(), "unable to set refresh token header", log.Error(err), log.ERROR)
```
Comment (lindenmckenzie) [inline directors/directors.go:78]: bad copy paste here

**Review by mr-nick17 — COMMENTED** [review]
Can you update the go version in `Dockerfile.local` to match the rest of the app? (I had to change it to get it working locally 😶) 

You might want to think about putting a feature flag in incase this gets to prod before the report works 😄

File: directors/directors.go (hunk 1 above)
Comment (jondewijones) [reply directors/directors.go]: 👍

[the remark this answers is not in the document]
Comment (jondewijones) [reply directors/directors.go]: 👍

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,8 @@
 module github.com/ONSdigital/florence
 
-go 1.20
+go 1.21
+
+toolchain go1.21.2
```
[the remark this answers is not in the document]
Comment (jondewijones) [reply go.mod]: 👍

**Review by mr-nick17 — DISMISSED** [review]

**Review by mr-nick17 — APPROVED** [review]

**Pull request merged**